command/test_utils.go

```diff
@@ -0,0 +1,62 @@
+package command
+
+import (
+	"bytes"
+	"testing"
+
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer/builder/file"
+	"github.com/hashicorp/packer/builder/null"
+	hcppackerimagedatasource "github.com/hashicorp/packer/datasource/hcp-packer-image"
+	hcppackeriterationdatasource "github.com/hashicorp/packer/datasource/hcp-packer-iteration"
+	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/post-processor/manifest"
+	shell_local_pp "github.com/hashicorp/packer/post-processor/shell-local"
+	filep "github.com/hashicorp/packer/provisioner/file"
+	"github.com/hashicorp/packer/provisioner/shell"
+	shell_local "github.com/hashicorp/packer/provisioner/shell-local"
+)
+
+// Utils to use in other tests
+
+// TestMetaFile creates a Meta object that includes a file builder
+func TestMetaFile(t *testing.T) Meta {
```

Comment:
Now that this is an exported function do you think plugin maintainers may want to pull in this pkg for testing? If so should we be concerned about any plugins pulling in Packer?

Reply:
Good question there, in an ideal future, Packer won't import any plugin; so that would be okay.
This has been quite handy to test things around in packerland. Maybe if this becomes an issue, we can move this pkg over in the SDK ? Or its own repo 🤔 .

Reply:
At the same time, if we don't document it, it probably won't be used much.